packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardTitle.tsx

```diff
@@ -137,14 +134,12 @@ export class DocumentCardTitle extends BaseComponent<IDocumentCardTitleProps, ID
       const { truncatedTitleFirstPiece, truncatedTitleSecondPiece } = this.state;
       this._isTruncated = true;
 
-      if (!truncatedTitleFirstPiece && !truncatedTitleSecondPiece) {
```

Comment:
Removing this block as `_shrinkTitle()` is called only on `componentDidUpdate()` when both pieces should have been already set in the `_startTruncation()` after mounting.